R/zzz.R

```diff
@@ -8,18 +8,16 @@
 
   # 1.2 Get available packages to see if each attempted to install is new
   current.packages <- as.data.frame(
-    available.packages(contriburl = contrib.url(repos = "https://cran.r-project.org/"))[, c(1, 2)],
+    available.packages(contriburl = contrib.url(repos = "https://cran.r-project.org/"), type = "binary")[, c(1, 2)],
```

Comment:
I'm not sure that's what we want since it won't work at all on Linux. I'm assuming the idea is to deal with the case when the package has been updated on CRAN but binaries have not yet been created. Why is this an issue?

Reply:
1. Does it actually come up empty on Linux? I am not sure if the  available.packages() commands restricts the results to the operating system requesting .
2. indeed it is needed because if the binary available is different, groundhog.library() may install a package that does not match the cran.toc.rds date, and which therefore differs across users. We don't want users to get a prompt asking for which version they want because (a) most users can't make sense of the question and (b) it breaks down reproducibility.
3. The goal was to only get CRAN if it will deliver the binary you need.  Would it make sense to just send Linux straight to install from source anyway? (from the tar.gz file we know where to find?)

Reply:
Ah! Surprisingly, this argument is just ignored on Linux so it's fine.